Vagrantfile

```diff
@@ -33,15 +38,15 @@ echo 'export GOPATH=#{gopath_folder}' > /etc/profile.d/envvar.sh
 echo 'export GOBIN=$GOPATH/bin' >> /etc/profile.d/envvar.sh
 echo 'export GOSRC=$GOPATH/src' >> /etc/profile.d/envvar.sh
 echo 'export PATH=$PATH:/usr/local/go/bin:$GOBIN' >> /etc/profile.d/envvar.sh
-echo "export http_proxy='$3'" >> /etc/profile.d/envvar.sh
-echo "export https_proxy='$4'" >> /etc/profile.d/envvar.sh
-echo "export USE_RELEASE=$5" >> /etc/profile.d/envvar.sh
-echo "export no_proxy=$2,127.0.0.1,localhost,netmaster" >> /etc/profile.d/envvar.sh
-echo "export CLUSTER_NODE_IPS=$2" >> /etc/profile.d/envvar.sh
-echo "export CONTIV_CLUSTER_STORE=$6" >> /etc/profile.d/envvar.sh
-echo "export CONTIV_V2PLUGIN_NAME=$9" >> /etc/profile.d/envvar.sh
-echo "export CONTIV_DOCKER_SWARM=${10}" >> /etc/profile.d/envvar.sh
-echo "export BUILD_VERSION=${11}" >> /etc/profile.d/envvar.sh
+echo "export http_proxy='#{http_proxy}'" >> /etc/profile.d/envvar.sh
```

Comment:
While you're cleaning this up, do you want to do something like:

```sh
cat >>/etc/profile.d/envvar.sh <<EOS
export FOO=foo
export BAR=bar
EOS
```

It'll cut the character count in half and make it easier to read :smile: